src/main/java/tick/tasks/Task.java

```diff
@@ -0,0 +1,78 @@
+package tick.tasks;
+
+/**
+ * Task class represents a task with a description and a status that indicates whether the task is done.
+ */
+public abstract class Task {
+    private String description;
+    private boolean isDone;

```

Comment:
i like the variable name `isDone`. Clearly describes it is a boolean variable representing if the task is done